### ISSUE TITLE
Ideas on balancing

### ISSUE THREAD

**Issue by LMH01** [Tondorf/AntServer#2]
I feel like the balancing could use some work. Killing ants is currently way to powerful as it gives 10 points whereas other methods of point generation only give one point.

Toxins are currently also quite weak, I think that they should at least retract 2 or 3 points from the enemy, especially if ant killing isn't nerfed. This would give the player more reason to try and defend against enemy ants that try to bring toxins to the own base.

What do you all think about this?

**Comment by r41d** [comment]
Hmm, killing ants is kind of special compared to getting one sugar, initially it gave much more points 🤔
About the toxin I'm not sure 🤔
What should be implemented next, I think, is the possibility to resurrect ants from the dead, either through sacrificing points or maybe even regularly?

**Comment by LMH01** [comment]
Yeah having ants resurrected would be a pretty good idea as the game gets slowed down immensely when only a few ants are left. This would also make it less valuable to instruct the own ants to kill all enemy ants as quickly as possible.

**Comment by r41d** [comment]
Maybe we could rather than patching things in-place build a definition of game mechanics with in-built balancing considerations 🤔

**Comment by LMH01** [comment]
Yeah we could do that. Or we create a proper config file where these balancing values can be changed. However I think that this would be too much to implement in this version, it might be something  that can be done when (or rather if) we rewrite the server in another language.
